### DISCUSSION TITLE
Conditionally use default props

### DISCUSSION THREAD

**Discussion by pisethx** [vuetifyjs/vuetify#12557] [Q&A]
I'm using v-data-table and I want to do something like this

```
<v-data-table
...
:custom-sort="backendSort ? onBackendSort : sortItems" // the default sortItems function 
></v-data-table>

```
is it possible to use the default function as a fallback value?

**Comment by MajesticPotatoe** [comment]
`:custom-sort="backendSort ? onBackendSort : undefined"`

**Reply by pisethx** [reply]
I tried it, it does not fallback to the default function at tall.
